src/content/reference/react/StrictMode.md

```diff
@@ -825,7 +827,19 @@ Without Strict Mode, it was easy to miss that your Effect needed cleanup. By run
 [Read more about implementing Effect cleanup.](/learn/synchronizing-with-effects#how-to-handle-the-effect-firing-twice-in-development)
 
 ---
+### Fixing bugs found by detaching and re-attaching DOM refs in development {/*fixing-bugs-found-by-cleaning-up-and-re-attaching-dom-refs-in-development*/}
```

Comment:
I think the `<Canary>` component should start above this line since this category is only applicable in canary

Reply:
it seems like the header doesn't show up properly in the outline if i do that. maybe can fix that though

Reply:
Not a big deal if it messes up the formatting

Reply:
We should just say "refs" instead of "DOM refs". Double invoking works for all of them now: https://ccp74m.csb.app/

Reply:
we don't currently have terminology to differentiate clearly between `ref`s as a general store for data (eg: a `ref` object created via `useRef` which points to some arbitrary value) versus `ref`s as passed to an element. we currently seem to conceptualize this in the docs as being centered around interation with DOM elements ([example](https://react.dev/learn/referencing-values-with-refs#refs-and-the-dom) even though this doesn't clearly encompass the kinds of refs like your example shows. open to suggestions though – perhaps i'm overthinking this and just saying "refs" is fine.

Reply:
what about "detaching and re-attachin refs to components" to make it clear we're not talking about refs as datastores like `useRef`?

Reply:
i think that makes sense. i wasn't sure if "components" would be inclusive of DOM elements, but it looks like elsewhere in the reference we refer to these as ["common components"](https://react.dev/reference/react-dom/components/common), so i think that works.